### PR TITLE
Implement entity models

### LINKED ISSUE #256: Create model - Fighter

### LINKED ISSUE #257: Create model - Ranger

### LINKED ISSUE #258: Create model - Knight

### LINKED ISSUE #259: Create model - Goblin

### LINKED ISSUE #260: Create model that can be used for the entity base
Create a model that can be used as the base of any entity. It should be primarily a darker color, but have white accents that can accept any color.

Think something like the bases in this image https://geekandsundry.com/wp-content/uploads/2016/05/CR-Minis.jpg

### PR DESCRIPTION
**Pull request by joshherkness**
- closes #256
- closes #257
- closes #258 
- closes #259 
- closes #260

### CODE REVIEW

**Comment by joshherkness** [comment]
Most of these file additions are simply the raw model data. No way to compress this,

**Comment by kdjohnson** [comment]
I'll look at this either later tonight or definitely tomorrow.

File: docker-compose.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ services:
     mongo:
         image: "mongo:latest"
         ports:
-            - "27017:27017"
+            - "27017:2017"
```
Comment (kdjohnson) [inline docker-compose.yml]: This needs to be changed back to `"27017:27017"`

File: vue/config/index.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = {
 
     // Various Dev Server settings
     host: 'localhost', // can be overwritten by process.env.HOST
-    port: 8080, // can be overwritten by process.env.PORT, if port is in use, a free one will be determined
+    port: 3000, // can be overwritten by process.env.PORT, if port is in use, a free one will be determined
```
Comment (kdjohnson) [inline vue/config/index.js:17]: Why was this port changed?

**Review by kdjohnson — CHANGES_REQUESTED** [review]

**Review by kdjohnson — APPROVED** [review]

**Pull request merged**